src/creature.cpp

```diff
@@ -688,13 +692,13 @@ BlockType_t Creature::blockHit(Creature* attacker, CombatType_t combatType, int3
 	}
 
 	if (attacker) {
-		if (Player* attackerPlayer = attacker->getPlayer()) {
+		if (auto attackerPlayer = attacker->getPlayer()) {
 			for (int32_t slot = CONST_SLOT_FIRST; slot <= CONST_SLOT_LAST; ++slot) {
 				if (!attackerPlayer->isItemAbilityEnabled(static_cast<slots_t>(slot))) {
 					continue;
 				}
 
-				Item* item = attackerPlayer->getInventoryItem(static_cast<slots_t>(slot));
+				auto item = attackerPlayer->getInventoryItem(static_cast<slots_t>(slot));
```

Comment:
Could this be `const auto`
```suggestion
				const auto item = attackerPlayer->getInventoryItem(static_cast<slots_t>(slot));
```